packages/modules/viewer-react/src/types.ts

```diff
@@ -1,3 +1,7 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
any idea why you keep getting duplicate headers?

Reply:
I think pre-commit automatically adds a header every time I commit.